bin/templates/scripts/cordova/defaults.xml

```diff
@@ -27,7 +27,7 @@
     <preference name="DisallowOverscroll" value="false" />
     <preference name="EnableViewportScale" value="false" />
     <preference name="KeyboardDisplayRequiresUserAction" value="true" />
-    <preference name="MediaPlaybackRequiresUserAction" value="false" />
+    <preference name="MediaTypesRequiringUserActionForPlayback" value="false" />
```

Comment:
This is for mediaTypesRequiringUserActionForPlayback which (already) included in other PR #784.
Or the AllowsAirPlayForMediaPlayback property here?

Reply:
Same as I mentioned before.